src/components/Results/StudyDetailsButton.tsx

```diff
@@ -15,8 +15,9 @@ const StudyDetailsButton = ({ icon, text, ...props }: StudyDetailsButtonProps):
       fontSize: '1.1em',
       fontWeight: '600',
       mb: 2,
-      minWidth: { xs: '100%', sm: '200px', xl: '400px' },
+      minWidth: { xs: '100%', sm: '200px' },
       width: '100%',
+      flex: 'none',
```

Comment:
This is to not make the buttons shrink when the facilities accordion is expanded